src/Tasks/Cleanup/Strategies/DefaultStrategy.php

```diff
@@ -99,10 +99,9 @@ protected function removeOldBackupsUntilUsingLessThanMaximumStorage(BackupCollec
             return;
         }
 
-        $maximumSize = $this->config->get('backup.cleanup.default_strategy.delete_oldest_backups_when_using_more_megabytes_than')
-            * 1024 * 1024;
+        $maximumSize = $this->config->get('backup.cleanup.default_strategy.delete_oldest_backups_when_using_more_megabytes_than');
 
-        if (!$maximumSize || ($backups->size() + $this->newestBackup->sizeInBytes()) <= $maximumSize) {
+        if ($maximumSize === null || ($backups->size() + $this->newestBackup->sizeInBytes()) <= $maximumSize * 1024 * 1024) {
```

Comment:
This is getting a bit unreadable. Could you extract this check to a separate function with a good name and early returns so we can avoid using `||`

Reply:
No problem